__tests__/app.test.js

```diff
@@ -7,295 +7,361 @@ const testData = require("../db/data/test-data/index");
+                    expect(comment).toHaveProperty("created_at");
+                    expect(comment).toHaveProperty("author");
+                    expect(comment).toHaveProperty("body");
+                    expect(comment).toHaveProperty("review_id");
+                });
+            });
+    });
+    test("200 but return an empty arr review found but no comments", () => {
+        return request(app)
+            .get("/api/reviews/1/comments")
+            .expect(200)
+            .then(({
+                body
+            }) => {
+                expect(body.comments).toHaveLength(0);
+            });
+    });
+    test("404 review cant be found", () => {
+        return request(app)
+            .get("/api/reviews/1000/comments")
+            .expect(404)
+            .then(({
+                body
+            }) => {
+                expect(body.msg).toBe("Sorry Review cant be found");
+            });
+    });
 });
 
-describe("Post add new comment review", () => {
-  test("201 new comment was added", () => {
-    const newComment = {
-      username: "mallionaire",
-      body: "I didn't know dogs could play games",
-    };
-    return request(app)
-      .post("/api/reviews/1/comments")
-      .send(newComment)
-      .expect(201)
-      .then(({ body }) => {
-        expect(body.comment).toHaveProperty("comment_id");
-        expect(body.comment).toHaveProperty("body");
-        expect(body.comment).toHaveProperty("votes");
-        expect(body.comment).toHaveProperty("author");
-        expect(body.comment).toHaveProperty("created_at");
-        expect(body.comment).toHaveProperty("review_id");
-      });
-  });
+describe("Get /api/reviews?sortby", () => {
+    test("200 returns all reviews with no query", () => {
+        return request(app)
+            .get("/api/reviews")
+            .expect(200)
+            .then(({
+                body
+            }) => {
+                expect(body.reviews).not.toHaveLength(0);
+                body.reviews.forEach((review) => {
+                    expect(review).toHaveProperty("review_id");
+                    expect(review).toHaveProperty("title");
+                    expect(review).toHaveProperty("designer");
+                    expect(review).toHaveProperty("owner");
+                    expect(review).toHaveProperty("review_img_url");
+                    expect(review).toHaveProperty("review_body");
+                    expect(review).toHaveProperty("category");
+                    expect(review).toHaveProperty("votes");
+                    expect(review).toHaveProperty("created_at");
+                });
+            });
+    });
+    test("sort reviews by date by default and asc by default", () => {
+        return request(app)
+            .get("/api/reviews")
+            .expect(200)
+            .then(({
+                body
+            }) => {
+                expect(body.reviews).toBeSortedBy("created_at");
+            });
+    });
```

Comment:
these two tests can be combined

The `GET` requests are the same and they're asserting similar things